### DISCUSSION TITLE
Release branches

### DISCUSSION THREAD

**Discussion by Kinrany** [arxanas/git-branchless#784] [Q&A] [answered]
How do you handle changes that need to go into multiple branches? Does it make sense to cherry-pick? Can git-branchless be used to later update both commits together?

**Comment by arxanas** [comment, marked as answer]
Cherry-picking is probably the best way to handle it. git-branchless uses the same "patch ID" mechanism as Git to detect patches which are duplicated across multiple branches, so if you were to `git move` or `git sync` one branch onto another, then git-branchless to decline to reapply the same patch.

There isn't currently a way to update two commits in different places with the same contents. See https://github.com/arxanas/git-branchless/discussions/553 for more discussion.
